addon/extend-qunit.js

```diff
@@ -3,6 +3,14 @@ import QUnit from 'qunit';
 
 const { assert } = QUnit;
 
+function failureMessage(base, custom) {
+  if (custom) {
+    return base + `: ${custom}`;
```

Comment:
What do you think about using string interpolation for all of this and omitting the `base + `?